cmd/riverui/logger.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"log/slog"
+	"os"
+	"strings"
+)
+
+var logger *slog.Logger //nolint:gochecknoglobals
+
+func initLogger() {
+	options := &slog.HandlerOptions{Level: getLogLevel()}
+	logger = slog.New(slog.NewTextHandler(os.Stdout, options))
+}
+
+func getLogLevel() slog.Level {
+	if isDebug() {
+		return slog.LevelDebug
+	}
```

Comment:
IMO let's just inline `isDebug`, it's longer overall extracted like this for just a 1-line gain at the callsite.

```suggestion
	debugEnv := os.Getenv("RIVER_DEBUG")
	if debugEnv == "1" || debugEnv == "true" {
		return slog.LevelDebug
	}
```